### PR TITLE
Add monodle model demos for Wormhole & Grayskull

### PR DESCRIPTION
**Pull request by ashokkumarkannan1**

### CODE REVIEW

File: model_demos/cv_demos/monodle/pytorch_monodle.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    pytorch_model = CenterNet3D(backbone="dla34")
+    pytorch_model.eval()
+
+    # Create pybuda.PyTorchModule using the loaded Pytorch model
+    tt_model = pybuda.PyTorchModule(model_name, pytorch_model)
+
+    # run inference on Tenstorrent device
+    output_q = pybuda.run_inference(tt_model, inputs=[(img_tensor,)])
+    output = output_q.get()[0].value()
+    print(output)
```
Comment (nvukobratTT) [inline model_demos/cv_demos/monodle/pytorch_monodle.py:58]: @milank94 can you provide some feedback on structuring this demo? :))

Comment (milank94) [reply model_demos/cv_demos/monodle/pytorch_monodle.py:55]: Looks like the output here will be the same at this: https://github.com/xingyizhou/CenterNet?tab=readme-ov-file#use-centernet

Can we confirm if this is the output: `ret will be a python dict: {category_id : [[x1, y1, x2, y2, score], ...], }`?

I'm thinking that the post-processing is coming from here: https://github.com/xingyizhou/CenterNet/blob/master/src/tools/vis_pred.py

If we can confirm that output schema is `{category_id : [[x1, y1, x2, y2, score], ...], }` or something like that, then we can just leave it as is. Or we can use that post-processing to generate the output image with annotations.

Comment (nvukobratTT) [reply model_demos/cv_demos/monodle/pytorch_monodle.py:55]: @ashokkumarkannan1 please provide more details when you find time :))

Comment (ashokkumarkannan1) [reply model_demos/cv_demos/monodle/pytorch_monodle.py:55]: Sure @nvukobratTT  . I will take a look into it today.

Comment (ashokkumarkannan1) [reply model_demos/cv_demos/monodle/pytorch_monodle.py:55]: Hi @milank94 , @nvukobratTT ,
I have researched the [CenterNet](https://github.com/xingyizhou/CenterNet?tab=readme-ov-file#use-centernet) for the postprocessing. But it seems [heads](https://github.com/xingyizhou/CenterNet/blob/4c50fd3a46bdf63dbf2082c5cbb3458d39579e6c/src/lib/opts.py#L302) used in CenterNet and Monodle seems different. So we can't use this post processing in Monodle model.

Comment (nvukobratTT) [reply model_demos/cv_demos/monodle/pytorch_monodle.py:55]: Thanks for the feedback @ashokkumarkannan1! 

@milank94 I'll merge these changes into the tracking branch. We need it merged in order to keep track of this model support on our nightly runs. We can discuss over the details before we release this model. I.e. do we need to add more modifications or not

**Review by nvukobratTT — APPROVED** [review]

**Pull request merged**